genai-perf/docs/tutorial.md

```diff
@@ -163,6 +163,17 @@ Example output:
 └───────────────────────────────────┴──────────┴──────────┴──────────┴──────────┴──────────┴──────────┘
 ```
 
+### Running with Authentication
```

Comment:
instead of having two separate (but same) authentication section in the tutorials, can we merge this into a single section and escalate it to our main README? I've also seen users needing authentications for other services (e.g. sagemaker) so I think this information to be useful in general.

Reply:
Sure, moved to the main README. Thanks!